cloud import: create a troubleshooting section (#14302)
pingcap/docs: tidb-cloud/import-csv-files.md

> **Note:**
>
> The URI must contain the data source folder.

## Troubleshooting

### Resolve warnings during data import

> After clicking **Start Import**, if you see a warning message such as "can't find the corresponding source files", resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.

Reviewer: ```suggestion
After clicking **Start Import**, if you see a warning message such as `can't find the corresponding source files`, resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes.
```

After resolving these issues, you need to import the data again.

### Zero rows in the imported tables

> After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched the value you entered in the **Source file name** field. In this case, check whether there are any typos in the **Source file name** field and try again.

Reviewer 2: ```suggestion
After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched in the Bucket URI that you entered. In this case,resolve this by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. And import those empty tables again.
```

Reviewer 2: @okJiang  PTAL

Reviewer 3: And import those empty tables again.

-> And import tables again.

Reviewer 3: rest LGTM

> After the import progress shows **Completed**, check the imported tables. If the number is zero, it means no data files matched in the Bucket URI that you entered. In this case, resolve this issue by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. After that, import those tables again.

Reviewer: ```suggestion
After the import progress shows **Completed**, check the imported tables. If the number of rows is zero, it means no data files matched the Bucket URI that you entered. In this case, resolve this issue by providing the correct source file, renaming the existing one according to [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md), or using **Advanced Settings** to make changes. After that, import those tables again.
```
